src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/EditorConfig/EditorConfigStorageLocation`1.cs

```diff
@@ -104,6 +105,18 @@ string IEditorConfigStorageLocation2.GetEditorConfigString(object? value, Option
             => $"{KeyName} = {((IEditorConfigStorageLocation2)this).GetEditorConfigStringValue(value, optionSet)}";
 
         string IEditorConfigStorageLocation2.GetEditorConfigStringValue(object? value, OptionSet optionSet)
```

Comment:
❔ there is a test covering this change?

Reply:
Yes, existing tests for "Configure code style" code fix failed with a cast exception here.